### PR TITLE
DM-1356 LWM2M device registration added to What's new

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
LWM2M device regs added to what's new!

### CODE REVIEW

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 1]
```diff
@@ -41,3 +41,21 @@ The Sigfox and Actility agents now support the configuration of multiple connect
...
+
+##### Single LWM2M device registration
+<video width="99%" controls>
+  <source src="/images/release-notes/lwm2m-reg.mp4" type="video/mp4" />
+</video>
+
+##### Bulk LWM2M device registration
+<video width="99%" controls>
+  <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
+</video>
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:61]:
```suggestion
</video>

For details, also refer to [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@sag-eweingaertner I added a link to the documentation. Please check if this is the correct place.

**Comment by sag-eweingaertner** [comment]
> @sag-eweingaertner I added a link to the documentation. Please check if this is the correct place.

Thank you for that. The link points to the right place in the documentation.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 2]
```diff
@@ -41,3 +41,21 @@ The Sigfox and Actility agents now support the configuration of multiple connect
...
+The video snippets below demonstrate these new wizards.
+
+##### Single LWM2M device registration
+<video width="99%" controls>
+  <source src="/images/release-notes/lwm2m-reg.mp4" type="video/mp4" />
+</video>
+
+##### Bulk LWM2M device registration
+<video width="99%" controls>
+  <source src="/images/release-notes/lwm2m-bulk-reg.mp4" type="video/mp4" />
```
Comment (Oezge-Akin) [inline content/release-10-15-0/whatsnew-10-15-0.md:60]: @sag-eweingaertner is it possible to crop the video to remove the desktop items?
Screenshot from 2022-08-15 09-55-55

**Review by Oezge-Akin — APPROVED** [review]

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 3]
```diff
@@ -41,3 +41,21 @@ The Sigfox and Actility agents now support the configuration of multiple connect
 ![All devices](/images/release-notes/multiple_lns_connection.png)
 
 For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
+
+### Device Management
+
+#### Revised LWM2M device registration
+
+The LWM2M agent now provides revised single- and device bulk registration wizards. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.
```
Comment (sourajitsaha17) [inline content/release-10-15-0/whatsnew-10-15-0.md:49]: single- and device bulk registration wizards -> single and bulk device registration wizards

Comment (sag-eweingaertner) [reply content/release-10-15-0/whatsnew-10-15-0.md]: fixed

**Pull request merged**